src/Symfony/Bridge/PhpUnit/SymfonyTestsListener.php

```diff
@@ -170,6 +172,13 @@ public function startTest(\PHPUnit_Framework_Test $test)
         }
     }
 
+    public function addWarning(\PHPUnit_Framework_Test $test, \PHPUnit_Framework_Warning $e, $time)
+    {
+        if ($test instanceof \PHPUnit_Framework_TestCase) {
```

Comment:
The type hint is for `\PHPUnit_Framework_Test`, but not `\PHPUnit_Framework_TestCase`.